### ISSUE TITLE
Passwords still exposed

### ISSUE THREAD

**Issue by DirkHoffmann** [indico/indico#1740]
Followup of #1530, unfortunately. There is another place in the source code which exposes passwords in the log messages.
I am still running v1.1.2, so I patched the logger myself to hide any `password` element of the HTTP parameters `dict`. However, in some places there is an additional element `passwordBis`. The log message goes then:
```
2014-10-02 12:43:03,462 indico.requestHandler - ERROR base.py:380 -- Request 3134164300 failed: "'NoneType' object has no attribute 'getAuthenticatorTag'"
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/indico-1.1.2-py2.7.egg/MaKaC/webinterface/rh/base.py", line 586, in process
    res = self._process()
  File "/usr/local/lib/python2.7/dist-packages/indico-1.1.2-py2.7.egg/MaKaC/webinterface/rh/users.py", line 160, in _process
    ih.add( id )
  File "/usr/local/lib/python2.7/dist-packages/indico-1.1.2-py2.7.egg/MaKaC/authentication/AuthenticationMgr.py", line 46, in add
    auth = self.getById( newId.getAuthenticatorTag() )
AttributeError: 'NoneType' object has no attribute 'getAuthenticatorTag'

Additional information:
URL: https://www.cta-observatory.org/indico/userRegistration.py
Params: {'lang': 'en_GB', 'fax': '123456', 'Save': 'confirm', 'surName': 'Timothyka', 'name': 'Timothyka', 'title': 'Dr.', 'organisation': 'SUNY Canton', 'telephone': '123456', 'passwordBis': 'nokUlEdot', 'displayTZMode': 'Event Timezone', 'address': 'ST', 'msg': '', 'timezone': 'Asia/Baghdad', 'login': 'Timothyka', 'password': '***', 'email': 'scottlwentwort at outlook.com'}
IP: 46.119.42.252
User Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1985.143 Safari/537.36 OPR/23.0.1522.77
Referer: http://www.cta-observatory.org/indico/userRegistration.py
``` 

Although this is an obvious spammer request, the `passwordBis` should not be exposed either. This is rather a one-liner, and I am almost tempted to submit the patch myself. Most difficult part is to use `git` correctly. :-}

**Comment by DirkHoffmann** [comment]
I am daring to submit code which has never been compiled (#1741). 
But I understand that there is a watchdog running on pull requests. So I am exploring. And discovering …

**Comment by ThiefMaster** [comment]
It's better to keep issues open even when you submit a PR for them. By including `fixes #1740` in the commit message or PR description it will be [closed automatically](https://help.github.com/articles/closing-issues-via-commit-messages/) once the PR (or the commit) is merged into master.

**Comment by pferreir** [comment]
(don't forget to backport it to 1.2)

**Comment by DirkHoffmann** [comment]
Today (10:46am) Pedro Ferreira wrote:

> 
> (don't forget to backport it to 1.2)

(Not sure, if I understand correctly.)

This is for your own bookkeeping, not for me, right?

**Comment by pferreir** [comment]
> This is for your own bookkeeping, not for me, right?

Yes, sorry, should have made it clearer :wink:

**Comment by pferreir** [comment]
Merged. Thanks!
